### DISCUSSION TITLE
low lighthouse performance scores

### DISCUSSION THREAD

**Discussion by s-kris** [chakra-ui/chakra-ui#5647] [Q&A]
I tried to build a very simple landing page with chakra-ui with nextjs and remix. 
The performance scores are low.

Deployment:
vercel

Performance score:
chakra + nextjs: 73 
chakra + remix: 78 
chakra + nextjs + js disabled: 94

links:
chakra+ nextjs: [https://chakra-nextjs-test.vercel.app](https://t.co/I3SjF36uFF)
chakra + remix: [https://chakra-remix-test.vercel.app](https://t.co/1GfMjclypP)
chakra+ nextjs + js disabled: [https://chakra-nextjs-test-no-js.vercel.app](https://t.co/D41xUTRpkw)

source code:
https://github.com/s-kris/chakra-nextjs-test
https://github.com/s-kris/chakra-remix-test

Can someone explain how to improve the scores without disabling the JS?
Thank you

**Comment by root0x** [comment]
Has anyone looked at this?

**Comment by hc0503** [comment]
I've checked your mention links with Lighthouse.
Could you explain about js disabled more clearly?
I don't understand what does that mean.
